framework/src/multiapps/MultiApp.C

```diff
@@ -516,22 +523,39 @@ MultiApp::fillPositions()
 void
 MultiApp::preTransfer(Real /*dt*/, Real target_time)
 {
-  // First, see if any Apps need to be Reset
-  if (!_reset_happened && target_time + 1e-14 >= _reset_time)
+  // Get a transient executioner to get a user-set tolerance
+  Real timestep_tol = 1e-13;
+  if (dynamic_cast<Transient *>(_fe_problem.getMooseApp().getExecutioner()))
+    timestep_tol =
+        dynamic_cast<Transient *>(_fe_problem.getMooseApp().getExecutioner())->timestepTol();
```

Comment:
Probably will need to include `Transient.h` here to do this `dynamic_cast`. Seeing a failure here: https://civet.inl.gov/job/1280286/

Reply:
ah yes